operations/update.go

```diff
@@ -53,96 +53,99 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
+			return DoUpdate(conf, ctx, doInstall, forceUpdate)
+		},
+	}
+}
 
-			client := conf.getRestCommunicator(ctx)
-			defer client.Close()
+func DoUpdate(conf *ClientSettings, ctx context.Context, doInstall bool, forceUpdate bool) error {
```

Comment:
Documentation.

I would rename this to `CheckAndUpdateVersion`.

Reply:
I would leave a doc comment for `CheckAndUpdateVersion` since it's recommended to document exported Go methods.

Reply:
I still have this open thread about documenting the function.